XenAdmin/TabPages/ManageUpdatesPage.cs

```diff
@@ -502,8 +516,7 @@ into g
         private void DeleteAllAllertAction_Completed(ActionBase sender)
         {
             Program.Invoke(Program.MainWindow, () =>
-            {                
-                Rebuild();

```

Comment:
As a sidenote : An interesting discovery I also made about the above is that when you dismiss all updates from another instance on XenCenter (without the changes), the message is displayed, and the column does not get resized at all